apps/files/src/default.js

```diff
@@ -33,14 +33,14 @@ const appInfo = {
       app: 'file-link',
       component: FileLink,
       enabled (capabilities) {
-        return capabilities.files_sharing.public.enabled === '1'
```

Comment:
Had to change the expected values here since the [changes](https://github.com/owncloud/owncloud-sdk/pull/345) in owncloud-sdk